src/style.css

```diff
@@ -0,0 +1,58 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars:not(:first-child) {
+  margin-top: -6px;
+}
+
+.stars--star {
+  width: 20px;
+  height: 20px;
```

Comment:
The star is actually 16x16 
please check the design
https://www.figma.com/file/EIBkG1dy1jnK88YPO34Qir/Moyo-%2F-Catalog-(updated)?node-id=11325%3A2961